storage/data-plane/src/test/java/com/microsoft/azure/storage/AppendBlobAPITest.groovy

```diff
@@ -23,7 +23,7 @@ class AppendBlobAPITest extends APISpec {
         bu.create(null, null, null, null).blockingGet()
     }
 
-    def "Create defaults"() {
+    def "AppendBlobAPITest Create defaults"() {
```

Comment:
There should be an alternative to append class name globally to method names when recording.
Visually, class name appends seem redundant everywhere here.

Reply:
I have raised the design of having a folder. However, after discussed with @jianghaolu , we decided to move forward with the simple change, and we will have better support for track 2. 

There are more than 30 tests need the change from 3 different classes. I do a regex replacement here to save eng time since it does not make too much difference change them as a whole or change them when necessary. I prefer to leave it as it is to save the resources.